src/routes/(auth)/inscription/+page.svelte

```diff
@@ -25,13 +26,15 @@
 			value={form?.password ?? ''}
 			name="password"
 			required
+			placeholder="**********"
 			type="password"
 			error={form?.password_error}>Mot de passe*</Input
```

Comment:
Required fields (*) should be specified by CSS

Reply:
I didn't succeed for that...
I would be of the opinion to leave it like this, not much time for those details maybe

Reply:
On the input's parent class add  :

```
&:has(input[required]) {
    label::after {
      content: "(*)";
    }
  }
```